psutil/_psutil_windows.c

```diff
@@ -2727,12 +2731,20 @@ psutil_proc_memory_maps(PyObject *self, PyObject *args) {
         if (GetMappedFileNameA(hProcess, baseAddress, mappedFileName,
                                sizeof(mappedFileName)))
         {
-            py_tuple = Py_BuildValue(
-                "(kssI)",
-                (unsigned long)baseAddress,
-                get_region_protection_string(basicInfo.Protect),
-                mappedFileName,
-                basicInfo.RegionSize);
+#ifdef _WIN64
+		   py_tuple = Py_BuildValue(
+			  "(KssI)",
+
+#else
+		   py_tuple = Py_BuildValue(
+			  "(kssI)",
+			  (unsigned long)baseAddress,
+#endif
+			  (unsigned long long)baseAddress,
```

Comment:
this looks wrong. Shouldn't this line be *inside* the ifdef statement?
You should do:

```


 +#ifdef _WIN64
 +         py_tuple = Py_BuildValue(
 +            "(KssI)",
 +            (unsigned long long)baseAddress,
 +#else
 +         py_tuple = Py_BuildValue(
 +            "(kssI)",
 +            (unsigned long)baseAddress,
 +#endif
              ...
```